README.md

```diff
@@ -1,3 +1,24 @@
+*NOTE*: Relativity.Transfer.Client version 7.4.10 (and below) will stop functioning on Feb 28th, 2023 due to migration of data transfer related products to cloud based solutions. The new version of Relativity.Transfer.SDK will be available on Nov 30th 2022.  
+
+**Since it is going to be a breaking change you will need to update your code and integrate with the new the new Relativity.Transfer.SDK version – 1.0.0.**

```

Comment:
We need to publish version 1.0.1 so maybe this version should be:
`SDK version – 1.0.*`
What do you think?

Reply:
what about the latest version

Reply:
Changed to 1.0.*